tests/lib/snaps/test-snapd-timedate-control-consumer/meta/snap.yaml

```diff
@@ -0,0 +1,13 @@
+name: test-snapd-timedate-control-consumer
+version: 1.0
+summary: Basic timedate-control consumer generic snap
+description: A basic generic snap that can be used to plug a timedate slot
+
+apps:
+    timedatectl:
+        command: bin/timedatectl
+        plugs: [] #timedate
```

Comment:
This syntax is confusing, what are you doing here? Neither of the two apps is using any plugs.

Reply:
Thanks for reviewing, the change is done.